docs/glossary.md

```diff
@@ -0,0 +1,72 @@
+
+### IntensityTable
+An intensity Table contains the features identified in an ImageStack. It can be thought of as an array whose entries are the intensities of each feature across the imaging rounds and channels of a field of view. Starfish exposes several processing tools to decode the features of the table, estimate their qualities, and assign features to cells. 
+
+### Codeword 
+A codeword maps expected intensities across multiple image tiles within a field of view to the target that is encoded by the codeword. 
+
+### Codebook
+A codebook contains all the codewords needed by an experiment to decode an IntensityTable. It also contains a mapping of channels to the integer indices that are used by starfish to represent them internally. 
+
+### Pipeline
+a sequence of data processing steps to process the inputs into the desired outputs.
+
+### Pipeline Component
+a single data processing step in the pipeline, as defined by its input and output file formats, e.g., the spot-detection component takes as input an image and outputs a table of spot locations, shapes, and intensities.
+
+### Pipeline Component Algorithm
+a specific algorithm type that adheres to the specified input and output file formats required by the component it belongs to. For example, a spot-detection component algorithm can be realized as a Gaussian blob detector or a connected components labeller. Both find spots and accept the same inputs and produce the same outputs, hence belong to the same component. However, the underlying properties of the algorithms (and parameterizations) may be quite different.
+
+### Pipeline Specification
+a document describing the above terms in detail, i.e., this document.
+
+### Pipeline Implementation
+actual code for the pipeline. This code will be packaged as a well-documented Python library and corresponding command line tool for use by consortium members to facilitate easy sharing and comparison of results across labs/methods.
+
+### Manifest
+The data manifest is a file that includes the locations of all fields of view for either primary or auxiliary images. 
+
+### Image Tile
+a single plane, single channel 2D image. In the manifest, each tile has information about it’s (X,Y,Z) coordinates in space, and information about which imaging round (R) and/or fluorescence channel (C) it was acquired under.
+
+### Field of View (FOV)
+a collection of Image Tiles corresponding to a specific volume or plane of the sample, under which the signal for all channels and all imaging rounds were acquired. All tiles within this FOV are the same size, but the manifest allows for different spatial coordinates for different imaging rounds or channels (to accommodate slight movement between rounds, for example).
+In microscopy, a field of view corresponds to the area of the camera sensor, and many such fields of view are expected to be taken per tissue slice. 
+
+### Coordinates (field of view): 
```

Comment:
The coordinates are actually per tile, which is correct in the Image Tile description, but a little confusing her.

Reply:
Ah, so the intuition would be that the coordinates should be able to capture jitter across Tiles in the field of view?